python/python/glide/async_commands/cluster_commands.py

```diff
@@ -415,6 +415,49 @@ async def function_delete(
             ),
         )
 
+    async def fcall_ro(
+        self,
+        function: str,
+        keys: Optional[List[str]] = None,
+        arguments: Optional[List[str]] = None,
+        route: Optional[Route] = None,
```

Comment:
This allows user to specify keys and route both. I think you need more than 1 function to restrict this. Alternative option is to do a runtime check and return a error.

Reply:
I have renamed cluster fcall_ro function to fcall_ro_route since Python doesn't support overloading.